### ISSUE TITLE
Argument descriptions are not formatting common mark.

### ISSUE THREAD

**Issue by ddebrunner** [magidoc-org/magidoc#186]
### Describe the bug

Common mark in field and directive argument descriptions is being ignored.

The argument description is displayed just as it is entered in the SDL file.

![image](https://user-images.githubusercontent.com/3769612/218332849-c3f23818-05c4-4c45-92c0-149f0e773fba.png)

(using 3.2.1)

### Reproduction

Simple SDL:
```
type Query {
  """
  this is *italics*, but this is `code`.
  """
  q(
    """
    is this is *italics*, but this is `code`.
    """
    a: Int
  ): Int
}
```

magidoc.mjs
```
export default {
  introspection: {
    type: 'sdl',
    paths: ['test.graphql'],
  },
  website: {
    output: 'docs',
    template: 'carbon-multi-page',
    options: {
      appTitle: 'A bug',
    },
  },
}
```

### System Info

```shell
System:
    OS: macOS 12.6.3
    CPU: (12) x64 Intel(R) Core(TM) i7-9750H CPU @ 2.60GHz
    Memory: 740.56 MB / 16.00 GB
    Shell: 5.8.1 - /bin/zsh
  Binaries:
    Node: 16.15.1 - /usr/local/bin/node
    Yarn: 1.22.18 - /usr/local/bin/yarn
    npm: 8.11.0 - /usr/local/bin/npm
  Browsers:
    Chrome: 109.0.5414.119
    Firefox: 86.0.1
    Safari: 16.2
```

### Severity

Annoyance

**Comment by pelletier197** [comment]
Hi! First of all, thank you for raising this issue and thank you for the level of details you got there. 

I know exactly what is the issue. I will implement it as soon as possible.

**Comment by pelletier197** [comment]
This is merged and release with version 3.3.0. Thank you for your contribution :slightly_smiling_face:
